### DISCUSSION TITLE
How to pack fyne app to `.deb` file?

### DISCUSSION THREAD

**Discussion by nchistov** [fyne-io/fyne#4612] [Q&A]
Now fyne apps packs to `package.tar.gz` on linux, but I want to know how can I pack it to `.deb` file.

It is easily to install, for a user without golang and fyne on his machine.

**Comment by Jacalz** [comment]
You can install the `.tar.xz` files using the provided Makefile without needing any Go compiler installed. As for `.deb`, it is not something I have seen used for Fyne apps yet. I suggest that you read up on how you create `.deb` packages in general. As long as you compile the binary and install the files to the same place as in our provided `Makefile` (see that tarball mentioned above) you should get a working package.
